test/Ssh.Tests.ps1

```diff
@@ -0,0 +1,12 @@
+. $PSScriptRoot\Shared.ps1
+
+Describe 'SSH Function Tests' {
+    Context 'Get-SshPath Tests' {
+        It 'Returns the correct default path' {
+            Get-SshPath | Should BeExactly $Home\.ssh\id_rsa
+        }
+        It 'Returns the correct path given a filename' {
+            Get-SshPath mykey | Should BeExactly $Home\.ssh\mykey

```

Comment:
OK so PowerShell internally handles both just fine but when you "compare exactly", the expected has to be correct.  Submitting a PR to fix this.